test/babashka/fs_test.clj

```diff
@@ -100,7 +100,12 @@
             sym-link (fs/create-sym-link (fs/file tmp-dir2 "sym-link") tmp-dir1)]
         (is (empty? (fs/glob sym-link "**")))
         (is (= 1 (count (fs/glob sym-link "**" {:follow-links true}))))
-        (is (= 1 (count (fs/glob (fs/real-path sym-link) "**")))))))
+        (is (= 1 (count (fs/glob (fs/real-path sym-link) "**"))))))
+    (testing ":hidden option"
```

Comment:
I kept this test non-Windows as the extra hassle to set a hidden attribute is worth it.